test/integration/local_transactions_test.rb

```diff
@@ -403,91 +405,180 @@ class LocalTransactionsTest < ActionDispatch::IntegrationTest
+      assert page.body.include?(content)
     end
 
-    context "without a custom button and link" do
-      setup do
-        stub_mapit_has_a_postcode_and_areas(
-          "WA8 8DX",
-          [0, 0],
-          [
-            {
-              "name" => "Cardiff",
-              "type" => "LGD",
-              "govuk_slug" => "cardiff",
-              "country_name" => "Wales",
-            },
-          ],
-        )
+    should "show a button that links to the local authority website" do
+      assert_has_button_as_link(
+        "Go to Cardiff website",
+        href: "http://cardiff.example.com",
+        rel: "external",
+      )
+    end
+  end
 
-        stub_local_links_manager_has_a_link(
-          authority_slug: "cardiff",
-          lgsl: 461,
-          lgil: 8,
-          url: "https://www.cardiff.gov.uk/bear-the-cost-of-grizzly-ownership",
-          country_name: "Wales",
-          status: "ok",
-        )
+  context "when a service is unavailable for a devolved administration without an interaction or homepage present" do
+    setup do
+      stub_mapit_has_a_postcode_and_areas(
+        "WA8 8DX",
+        [0, 0],
+        [
+          {
+            "name" => "Cardiff",
+            "type" => "LGD",
+            "govuk_slug" => "cardiff",
+            "country_name" => "Wales",
+          },
+        ],
+      )
 
-        visit "/pay-bear-tax"
-        fill_in "postcode", with: "WA8 8DX"
-        click_on "Find"
-      end
+      stub_local_links_manager_has_no_link_and_no_homepage_url(
+        authority_slug: "cardiff",
+        lgsl: 461,
+        lgil: 8,
+        country_name: "Wales",
+      )
 
-      should "render the service unavailable in country page" do
-        assert page.has_content? "Pay your bear tax"
-      end
+      visit "/pay-bear-tax"
+      fill_in "postcode", with: "WA8 8DX"
+      click_on "Find"
+    end
 
-      should "show a button that links to an appropriate alternate service provider" do
-        assert_has_button_as_link(
-          "Find other services",
-          href: "http://cardiff.example.com", # local authority link from stubbed local links
-          rel: "external",
-          start: true,
-        )
-      end
+    should "render unavailable_service view" do
+      assert page.has_content? "This service is not available in Wales."
+    end
+
+    should "render matching the postcode to local authority" do
+      content = "We've matched the postcode to <span class=\"local-authority\">Cardiff</span>."
+      assert page.body.include?(content)
+    end
+
+    should "not link to the authority" do
+      assert page.has_no_link?("Go to their website")
+    end
+
+    should "show advisory message that we have no url" do
+      assert page.has_content?("We don't have a link for their website.")
+      assert page.has_link?("local council search", href: "/find-local-council")
+    end
+  end
+
+  context "when a service is available for a devolved administration" do
```

Comment:
```suggestion
  context "when a service is handled differently for a devolved administration" do
```

I think we have to be careful we don't make this sound like the normal situation but for a devolved administration postcode.

Reply:
👍